types/src/access_rights.rs

```diff
@@ -11,6 +12,7 @@ bitflags! {
     /// A struct which behaves like a set of bitflags to define access rights associated with a
     /// [`URef`](crate::URef).
     #[allow(clippy::derive_hash_xor_eq)]
+    #[derive(Deserialize, Serialize)]
```

Comment:
We don't need this change do we?

Reply:
Ah, that was to allow serialization of Bid (bonding_purse:URef), but ended up creating a json_compatibility struct for Bid instead. Will remove.

Reply:
Did this.